components/SearchForm/index.js

```diff
@@ -1,8 +1,66 @@
 import React, { useState } from "react";
 import { employees } from "@/test_daten/test_daten";
+import styled from "styled-components";
 
+// Styled Components
+const SearchBarContainer = styled.form`
+  display: flex;
+  align-items: center;
+`;
+
+const SearchInput = styled.input`
+  margin-right: 10px;
+`;
+
+const SearchButton = styled.button`
+  background-color: gold;
+  color: black;
+  border: none;
+  padding: 5px 10px;
+  cursor: pointer;
+`;
+
+const ShowAllButton = styled.button`
+  background-color: gold;
+  color: black;
+  border: none;
+  padding: 5px 10px;
+  cursor: pointer;
+`;
+
+const SortDropdownContainer = styled.div`
+  margin-top: 10px;
+`;
+
+const EmployeeList = styled.ul`
+  list-style: none; /* Entfernt die Aufzählungspunkte */
+`;
+
+const EmployeeListItem = styled.li``;
```

Comment:
same here, if the component is empty just use the element itself in the index.js